.dagger/config/run_debug.sh

```
@@ -0,0 +1,37 @@
+envFile="/envFile"
+
+# Check if the file exists
+if [ ! -f "$envFile" ]; then
+  echo "Environment file not found: $envFile"
+  exit 1
+fi
+
+# Read the file and export each line as an environment variable
+while IFS='=' read -r key value; do
+  # Ignore empty lines or comments
+  if [ -z "$key" ] || [ "${key:0:1}" = "#" ]; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, string indexing is undefined.

(SC3057)